src/components/sectionLouka/Louka.tsx

```diff
@@ -0,0 +1,87 @@
+import { useLocation } from "react-router-dom";
+import { ROUTES } from "../../routes";
+import { ButtonLink } from "../ButtonLink";
+
+export const Louka = () => {
```

Comment:
kdyz odhlidnu od toho, ze nechapu co to je, tak je to trikrat to stejny = les, louka, park. Kdyz uz tak byt to mela byt jedna komponenta obecna co dostane data.